be/src/exec/olap_scan_node.cpp

```diff
@@ -653,6 +656,30 @@ Status OlapScanNode::normalize_conjuncts() {
     return Status::OK();
 }
 
+static std::string olap_filter_to_string(const doris::TCondition& condition) {
+    std::stringstream ss;
```

Comment:
better use `format` to do thing of format output. `std::stringstream` have performance problem in multi thread